src/client/pages/Landing/Landing.tsx

```diff
@@ -181,8 +281,8 @@ const LandingPageCardsSe: React.FC<{
   return (
     <>
       <Card to={`/${language}/new-member/new`}>
-        <Headline>{textKeys.STARTPAGE_UNINSURED_HEADLINE()}</Headline>
-        <Paragraph>{textKeys.STARTPAGE_UNINSURED_BODY()}</Paragraph>
+        <CardHeadline>{textKeys.STARTPAGE_UNINSURED_HEADLINE()}</CardHeadline>
```

Comment:
Imo this file is a bit large and therefore hard to get an overview of - maybe some things, like for example the `LandingPageCardsSe` component could be in its own file? With that said, I don't think we have to do this now. Just something to maybe fix in the future.

Reply:
True that! 🙌 How do we feel about having separate files for styling btw? I'm used to it myself but either or works. And maybe just in cases where you end up with large files.
Pros: shorter files and clearer separation
Cons: more files and imports

Reply:
I think in this case if we break out some of the components some styled components will follow, like for instance Headline, Preamle, Wrapper and UspContainer are only used for one part, that could be split out into a separate file.
 
If we split out the different LandingPageCards [se,no,dk] I think the common Styled components used by those should be in a separate file.